peer/store-utils-mock.go

```diff
@@ -0,0 +1,19 @@
+package peer
+
+// This file contains mock functions to be used during unit tests.
```

Comment:
Then this probably doesn't belong here in package `peer` and should be in package `tests` ?